CMakeLists.txt

```diff
@@ -205,6 +205,8 @@ if(EMSCRIPTEN)
   add_subdirectory(webassembly)
 endif ()
 
+option(F3D_USE_EXTERNAL_DEPS "Use external dependencies" OFF)
```

Comment:
Would be more convenient if we had two different options `F3D_USE_EXTERNAL_CXXOPTS` and `F3D_USE_EXTERNAL_NLOHMANN_JSON`.

Reply:
Thank for the suggestion, I'll use these two options later.